ui/src/main/java/com/kumuluz/ee/openapi/mp/ui/filters/SwaggerUIFilter.java

```diff
@@ -35,15 +58,30 @@
  */
 public class SwaggerUIFilter implements Filter {
 
+    private static final Logger LOG = Logger.getLogger(SwaggerUIFilter.class.getName());
+
+
+    private static final String X_ORIGINAL_URI = "X-Original-URI";
+    private static final String OAUTH_QUERY = "&oauth2RedirectUrl=";
+    private static final String URL_QUERY = "/?url=";
+    private static final List<String> LOCAL_HOST_ALIASES = Arrays.asList("localhost", "loopback", "127.0.0.1", "[::1]");
+
+
     private String specUrl;
     private String uiPath;
     private String oauth2RedirectUrl;
+    private String specPath;
+    private boolean isAutoConfigure;
+    private boolean isServerListUpdateEnabled;
```

Comment:
To simplify configuration I propose we only use two properties - isAutoConfigure and isXURICheckEnabled. The server list should be updated always when autoConfigure is enabled - even if the correct URL is already listed it should be moved to the top (if not already).

Reply:
done